https://gitlab.com/kwinft/kwinft/-/issues/209

CMake Error at CMakeLists.txt:816 (add_executable):
  Target "kwin_wayland" links to target "Threads::Threads" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?

--- CMakeLists.txt.orig	2021-10-14 15:22:02 UTC
+++ CMakeLists.txt
@@ -168,6 +168,12 @@ set_package_properties(XKB PROPERTIES
 
 find_package(Qt5XkbCommonSupport REQUIRED)
 
+find_package(Threads)
+set_package_properties(Threads PROPERTIES
+    TYPE REQUIRED
+    PURPOSE "Required for building KWin with Wayland support"
+)
+
 find_package(Libinput 1.9)
 set_package_properties(Libinput PROPERTIES TYPE REQUIRED PURPOSE "Required for input handling on Wayland.")
 
